functions/homework_function_is_prime.py

```diff
@@ -0,0 +1,46 @@
+"""Solution of task 2"""
+
+
+def is_prime(number):
+    if number not in range(2, 1001):
+        return f"your entered number is out of range(2, 1000)"
+
+    for num in range(2, int(number ** 0.5) + 1):
+        if number % num == 0:
+            return False
+
+    return True
+
+
+def check_correct_input_number():
+    while True:
+        test_number = input(
+            "Please, enter any number in range from 2 to 1000 to check whether it is prime or composite: ")
+        try:
+            checked_number = int(test_number)
+
+        except ValueError:
+            print("You entered incorrect value. You should enter integer number.")
+
+        else:
+            print(f"You entered: {checked_number}")
+            break
+
+    return checked_number
```

Comment:
def validate_datatype(variable, expected_type):
    if isinstance(variable, expected_type):
        return variable
    else:
        raise ValueError(f'Expected datatype for variable {variable} is {expected_type}.'
                         f'\n Actual is {type(variable)}'
                         f'\nTry again')